### PR TITLE
Fix Documentation: fix broken link in getting-started page

### PR DESCRIPTION
**Pull request by rafedramzi**
Hi! It's that 1 liner fix broken link in the docs PR. It bugs me when I found a broken link in the docs, so here's the fix (tested locally against your master branch). I also think you need to cherry-pick this commit into your other release branches as well (maybe v4.xx release?), cheers!

### CODE REVIEW

**Review by Nokel81 — APPROVED** [review]

File: docs/getting-started/README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ Lens is lightweight and simple to install. You'll be up and running in just a fe
 
 ## System Requirements
 
-Review the [System Requirements](/supporting/requirements/) to check if your computer configuration is supported.
```
Comment (jim-docker) [inline docs/getting-started/README.md:8]: this seems to work when the docs are served locally...

File: docs/getting-started/README.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ Lens is lightweight and simple to install. You'll be up and running in just a fe
...
+Review the [System Requirements](../supporting/requirements.md) to check if your computer configuration is supported.
```
Comment (jim-docker) [inline docs/getting-started/README.md:8]: ...but this is in line with other links in this file anyway

**Review by jim-docker — APPROVED** [review]

File: docs/getting-started/README.md (hunk 2 above)
Comment (Nokel81) [reply docs/getting-started/README.md:8]: Ah so it should be this instead?

Comment (jim-docker) [reply docs/getting-started/README.md:8]: ? This is what @rafedramzi proposed, I was merely agreeing it is suitable, even though the original link worked locally.

**Comment by rafedramzi** [comment]
Ok I should probably explain the problem more clearly:

When you're testing locally, the URL is http://localhost:8000/getting-started/ so
/supporting/requirements/ will be evaluated as http://localhost:8000/supporting/requirements/ looks good so far


But on your prod, the URL is https://docs.k8slens.dev/v4.1.1/getting-started/ ans also it has multiple version deployed!
So /supporting/requirements/ will be evaluated as https://docs.k8slens.dev/supporting/requirements/ and that's not correct
The correct URL should be https://docs.k8slens.dev/v4.1.1/supporting/requirements/

Edit: wrong link evaluation, just fixed it

File: docs/getting-started/README.md (hunk 2 above)
Comment (rafedramzi) [reply docs/getting-started/README.md:8]: Sorry I didn't explain the problem, I explain the problem in https://github.com/lensapp/lens/pull/2155#issuecomment-781393646

**Pull request merged**